src/test/java/com/stripe/model/AccountTest.java

```diff
@@ -1,195 +1,19 @@
-    String json = resource("account.json");
-    stubNetwork(Account.class, json);
-    final Account acc = Account.retrieve("acct_1032D82eZvKYlo2C");
-    verifyGet(Account.class, "https://api.stripe.com/v1/accounts/acct_1032D82eZvKYlo2C");
-
-    Map<String, Object> params = new HashMap<String, Object>();
-    Map<String, Object> legalEntity = new HashMap<String, Object>();
-    legalEntity.put("type", "individual");
-    params.put("legal_entity", legalEntity);
-    acc.update(params);
-
-    verifyPost(Account.class, "https://api.stripe.com/v1/accounts/acct_1032D82eZvKYlo2C", params);
-    verifyNoMoreInteractions(networkMock);
-  }
-
-  @Test
-  public void testAccountCreateExternalAccount() throws StripeException, IOException {
-    String json = resource("account.json");
-    stubNetwork(Account.class, json);
-    final Account acc = Account.retrieve("acct_1032D82eZvKYlo2C");
-    verifyGet(Account.class, "https://api.stripe.com/v1/accounts/acct_1032D82eZvKYlo2C");
-
-    Map<String, Object> params = new HashMap<String, Object>();
-    params.put("external_account", "tok_1234");
-    stubNetwork(Card.class, "{\"id\": \"tok_1234\", \"object\": \"card\", \"last4\": \"1234\"}");
-    ExternalAccount ea = acc.getExternalAccounts().create(params);
-
-    assertEquals("card", ea.getObject());
-    assertEquals("1234", ((Card) ea).getLast4());
-
-    verifyPost(ExternalAccount.class, "https://api.stripe.com/v1/accounts/acct_1032D82eZvKYlo2C/external_accounts", params);
-    verifyNoMoreInteractions(networkMock);
-  }
-
-  @Test
-  public void testAccountDelete() throws StripeException, IOException {
-    String json = resource("account.json");
-    stubNetwork(Account.class, json);
-    Account acc = Account.retrieve("acct_1032D82eZvKYlo2C");
-    verifyGet(Account.class, "https://api.stripe.com/v1/accounts/acct_1032D82eZvKYlo2C");
-
-    stubNetwork(DeletedAccount.class, "{\"id\": \"acct_1032D82eZvKYlo2C\", \"deleted\": \"true\"}");
-    acc.delete();
-    verifyDelete(DeletedAccount.class, "https://api.stripe.com/v1/accounts/acct_1032D82eZvKYlo2C");
-
-    verifyNoMoreInteractions(networkMock);
-  }
-
-  @Test
-  public void testAccountCreateLoginLink() throws StripeException, IOException {
-    String json = resource("account_express.json");
-    stubNetwork(Account.class, json);
-    Account acc = Account.retrieve("acct_EXPRESS");
-    verifyGet(Account.class, "https://api.stripe.com/v1/accounts/acct_EXPRESS");
-
-    String jsonLink = resource("login_link.json");
-    stubNetwork(LoginLink.class, jsonLink);
-    acc.getLoginLinks().create();
-    verifyPost(LoginLink.class, "https://api.stripe.com/v1/accounts/acct_EXPRESS/login_links", (RequestOptions) null);
+  public void testDeserialize() throws Exception {
+    String data = getFixture("/v1/accounts/acct_123");
+    Account resource = APIResource.GSON.fromJson(data, Account.class);
+    assertNotNull(resource);
+    assertNotNull(resource.getId());
```

Comment:
we likely want specific edge-case as fixtures to ensure deserialization works as needed.